libs/java/cert_refresher/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.yahoo.athenz</groupId>
+        <artifactId>athenz</artifactId>
+        <version>1.7.26-SNAPSHOT</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>athenz-cert-refresher</artifactId>
+    <name>athenz-cert-refresher</name>
+    <description>Athenz SSL Cert Refresher Package</description>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.jmockit</groupId>
+            <artifactId>jmockit</artifactId>
+            <version>1.32</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
just bringing guava dependency for java.io.Resources is a bit overkill in my opinion. You should use the classloader's get resource as a stream method instead or scope this at least to test only.

Reply:
did the easiest thing and made it scope test ;)